### DISCUSSION TITLE
Where is the Dockerfile for arigaio/atlas?

### DISCUSSION THREAD

**Discussion by ClaireNeveu** [ariga/atlas#805] [Q&A]
I'm trying to use the arigaio/atlas image with Google Cloud Build and having some issues there. I went to look for the Dockerfile to see which base image it's using as its base but I can't seem to locate it.

**Comment by rotemtam** [comment]
Hi @ClaireNeveu,

The official Atlas distribution is built internally from a private repository in Ariga that contains all of our deployments. Happy to share it here with you:

```Dockerfile
# This is a minimal image that contains all of the packages in gcr.io/distroless/static + glibc, libssl, openssl
FROM gcr.io/distroless/base-debian11

ARG ATLAS_EXEC=/atlas
ARG TARGETARCH
COPY --chmod=001 $TARGETARCH/$ATLAS_EXEC /atlas

ENTRYPOINT ["/atlas"]
```


If you want to create a Docker image from source, you can use a docker file similar to this:

```Dockerfile
FROM golang:1.18 as build

RUN go install ariga.io/atlas/cmd/atlas@master

FROM gcr.io/distroless/base-debian11

COPY --from=0 /go/bin/atlas /atlas

ENTRYPOINT ["/atlas"]
```

I'm available on our [Discord](https://discord.gg/zZ6sWVg6NT) server, I'll be happy to help you debug whatever you are facing.
